Pipfile

```diff
@@ -14,4 +14,4 @@ python_version = "3"
 
 [packages]
 scons = "*"
-scons-parts = "*"
+scons-parts = "==v0.17.0"
```

Comment:
Can you commit this separately, please? It would be nice to have a single commit to revert when Jason cuts a release with the fix.

Reply:
Good point. Actually I reverted this as PV builds successfully without it. Maybe Jason has cut a new release already.

Reply:
Cool